### ISSUE TITLE
Disabling Secure Access disables registry access options

### ISSUE THREAD

**Issue by stuclem** [vmware/vic-ui#250]
If you toggle **Enable secure access to this VCH** to OFF, this also deactivates the Registry Access tab.

However, in the CLI docs for the `--registry-ca` option, we state the following:

---

The use of registry certificates is independent of the Docker client security options that you specify. For example, it is possible to use the `--no-tls` option to disable TLS authentication between Docker clients and the VCH, and to use the `--registry-ca` option to enable TLS authentication between the VCH and a private registry.

---

Basically, if you toggle **Enable secure access to this VCH** to OFF, you can't use this VCH with VIC Registry.

**Comment by cristianfalcone** [comment]
@lweitzman @jak-atx @zjs any thoughts on this?

**Comment by jak-atx** [comment]
We need to know what the desired behavior is here.

cc @mhagen-vmware @mlh78750 @mdubya66 for triage

**Comment by zjs** [comment]
The current UX is intentional. We can revisit in the future, but this definitely doesn't seem like a ship-stopper.

By disabling the slider for client certificates, instead on the slider at the top of the page, the user can configure registry settings without configuring TLS verification.

**Comment by stuclem** [comment]
Fair enough. Hadn't experimented with doing it that way.

**Comment by jak-atx** [comment]
Closing as addressed in https://github.com/vmware/vic-ui/issues/296
